Formula/util-linux.rb

```diff
@@ -43,6 +43,21 @@ def install
     end
   end
 
+  def caveats
+    <<~EOS
+      The following tools are not supported under macOS, and are therefore not included:
+        addpart agetty blkdiscard blkzone blockdev chcpu chmem choom chrt ctrlaltdel delpart
+        dmesg eject fallocate fdformat fincore findmnt fsck fsfreeze fstrim hwclock ionice
+        ipcrm ipcs kill last ldattach losetup lsblk lscpu lsipc lslocks lslogins lsmem lsns
+        mount mountpoint nsenter partx pivot_root prlimit raw readprofile resizepart rfkill
+        rtcwake script scriptlive setarch setterm sulogin swapoff swapon switch_root taskset
+        umount unshare utmpdump uuidd wall wdctl zramctl
```

Comment:
Could we print this based on an array? It's difficult to read in code.

Reply:
Do you mean something like this?
```
    linux_only = %w[
      addpart agetty
      blkdiscard blkzone blockdev
      ...
      wall wdctl
      zramctl
    ]
    system_bins = %w[
      cal col colcrt colrm
      getopt
      ...
      ul
      whereis
    ]
    <<~EOS
      The following tools are not supported under macOS, and are therefore not included:
      #{Formatter.wrap(Formatter.columns(linux_only), 80)}
      The following tools are already shipped by macOS, and are therefore not included:
      #{Formatter.wrap(Formatter.columns(system_bins), 80)}
    EOS
```
That would output like this:
```
The following tools are not supported under macOS, and are therefore not included:
addpart      blkzone      chmem        ctrlaltdel   eject        fincore     
fsfreeze     ionice       kill         losetup      lsipc        lsmem       
...
rfkill       scriptlive   sulogin      switch_root  unshare      wall


The following tools are already shipped by macOS, and are therefore not included:
cal               col               colcrt            colrm             getopt  
...
nologin           renice            rev               ul                whereis
```

Reply:
That looks even better than I was thinking.

Reply:
Also, the contents of `system_bins` above is copied from another array in the `install` method. Instead of maintaining two separate lists that may change over time, is there a way to define a single array that can be shared across the `install` and `caveats` methods?

Defining a class variable `@@system_bins` works, but `brew style` says to define a class instance variable instead, and [my feeble attempt seems to expand to an empty string](https://gist.github.com/gromgit/4afbb98979bf83bd21614e57fd1b8107). I haven't found an existing formula that uses instance variables, so do you have any suggestions?

Reply:
You could possibly define a function? I think style will be ok with that.

Reply:
Yeah, we have formulae that define functions for shared stuff